config/test.json

```diff
@@ -12,6 +12,7 @@
     "port": "8080"
   },
   "mapproxy": {
+    "fileProvider": "fs",
```

Comment:
open issue to update the helm chart

Reply:
fixed: https://rnd-hub.atlassian.net/browse/MAPCO-439